### ISSUE TITLE
pyenv: still trying to use `pathman`

### ISSUE THREAD

**Issue by coolaj86** [webinstall/webi-installers#697]
- [x] pathman
- [ ] liblzma-dev

```sh
webi python3
```

```text
pyenv-bootstrap.sh: 570: pathman: not found
```

**Comment by coolaj86** [comment]
How embarrassing... it's systemic

```text
bun/bun-install.bash
13:    pathman add "$HOME/.bun/bin" > /dev/null 2>&1

pathman/README.md
29:pathman add ~/.local/bin

rustlang/install.sh
12:    pathman add "$HOME/.cargo/bin"

node/install.bat
16:  pathman add .local\opt\node-v12.16.2 || goto :error

pyenv/install.sh
48:    pathman add ~/.pyenv/bin
51:    pathman add ~/.pyenv/shims

gpg/install.sh
37:    pathman add ~/.local/opt/gnupg/bin

_webi/bootstrap.bat
19:    rem pathman add "%userprofile%\.local\bin" >nul 2>&1 || goto :error
20:    pathman add "%userprofile%\.local\bin" || goto :error
```

**Comment by coolaj86** [comment]
Closed via #702 (in hotfix)
